modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/DataSet.js

```diff
@@ -374,7 +390,13 @@ const DataSet = ({
 						{...currentViewProps}
 					/>
 				) : (
-					<EmptyResultMessage />
+					<ClayEmptyState
+						description={Liferay.Language.get(
+							'no-items-were-found'
+						)}
+						imgSrc="/o/classic-theme/images/states/empty_state.gif"
+						title=""
```

Comment:
Looking at existing usages of `ClayEmptyState` in DXP, we are consistently using `title={Liferay.Language.get('no-results-found')}`. Why not add it here? `EmptyResultMessage` does not have it, but that is not following Lexicon guidelines, as @drakonux mentioned in [the Slack discussion](https://liferay.slack.com/archives/CNBG06JS3/p1648550857756109). One of our goals with FDS is to make it behave and look more in line with Lexicon.

/cc @matuzalemsteles

Reply:
Yes, I agree with this

Reply:
Hey guys, if you can do that it will be great but if that is not possible we can leave this API as optional if it passes `null`, Diego sent a PR for this https://github.com/liferay/clay/pull/4779. Let me know if you guys go with the Lexicon recommendation.

Reply:
Hey @matuzalemsteles 
Finally we go for following Lexicon standards in this case.
However, I think is not a bad idea to dismiss the title markup if no title is present. Proceed as you think is better 👌